NWP - extend constant parameter documentation (#410)
MeteoSwiss/opendata: docs/e-forecast-data/e2-e3-numerical-weather-forecasting-model.mdx

:::
### Available parameters {#available-parameters}

Each collection contains both **constant** and **variable parameters**.

> **Variable parameters** change over time and include quantities such as air pressure and temperature. <br />

Reviewer: ```suggestion
**Variable parameters** change over time and include forecast quantities such as air pressure and temperature. <br />
```

**Constant parameters** remain fixed within a single model run and describe the underlying grid or surface properties. We distinguish between the following categories.

- Horizontal constants:

>     - **HHL** Geometric Height of the layer limits above sea level, see the model grid documentation

Reviewer: ```suggestion
    - **HHL** Geometric height of the layer limits above sea level, see the [model grid documentation](/e-forecast-data/e2-e3-numerical-weather-forecasting-model#vertical-grid)
```

- Vertical constants: 
    - **CLON**	Longitude of the center point coordinate of each triangle on the horizontal grid
    - **CLAT**	Latitude of the center point coordinate of each triangle on the horizontal grid
    - **FR_LAND**	Land cover indicator (1 for land, 0 for sea)
    - **HSURF**	Geometric height of the earth's surface above sea level
    - **SOILTYPE**	Type of soil, based on the [soilType.table](https://github.com/COSMO-ORG/eccodes-cosmo-resources/blob/master/definitions/soilType.table)

For a practical example of how to download constant parameters in Python, refer to notebook [09_constant_parameters.ipynb](https://github.com/MeteoSwiss/opendata-nwp-demos/blob/main/09_constant_parameters.ipynb).

:::info

> When downloading variable parameters in Python, the corresponding constant parameters (CLON, CLAT) are automatically included. 

Reviewer: ```suggestion
When downloading variable parameters through the Python API, the corresponding horizontal constant parameters (CLON, CLAT) are automatically included. 
```
HHL is not included, right?

Raw forecast GRIB files loaded via the REST API do not contain information about height, longitude and latitude.
To geolocate or interpret vertical levels, you must use the static vertical and horizontal grid parameter files provided in each collection.

> For more information, refere to notebook [09_constant_parameters.ipynb](https://github.com/MeteoSwiss/opendata-nwp-demos/blob/main/09_constant_parameters.ipynb) or read section [Accessing static grid information: Height, longitude, latitude and surface properties](/e-forecast-data/e2-e3-numerical-weather-forecasting-model?download-options=restapi#accessing-static-grid-information-height-longitude-latitude-and-surface-properties).

Reviewer: ```suggestion
For more information, refer to notebook [09_constant_parameters.ipynb](https://github.com/MeteoSwiss/opendata-nwp-demos/blob/main/09_constant_parameters.ipynb) or read section [Accessing static grid information: Height, longitude, latitude and surface properties](/e-forecast-data/e2-e3-numerical-weather-forecasting-model?download-options=restapi#accessing-static-grid-information-height-longitude-latitude-and-surface-properties).
```
